docs/cloud-storage-fileset-example.md

```diff
@@ -0,0 +1,678 @@
+---
+title: "How to use cloud storage fileset"
+slug: /how-to-use-cloud-storage-fileset
+keyword: fileset S3 GCS ADLS OSS
+license: "This software is licensed under the Apache License version 2."
+---
+
+This document aims to provide a comprehensive guide on how to use cloud storage fileset created by Gravitino, it usually contains the following sections:
+
+## Necessary steps in Gravitino server
+
+### Start up Gravitino server
+
+Before running the Gravitino server, you need to put the following jars into the fileset catalog classpath located at `${GRAVITINO_HOME}/catalogs/hadoop/libs`. For example, if you are using S3, you need to put `gravitino-aws-hadoop-bundle-{gravitino-version}.jar` into the fileset catalog classpath in `${GRAVITINO_HOME}/catalogs/hadoop/libs`.
+
+| Storage type | Description                                                   | Jar file                                                                                                                 | Since Version    |
+|--------------|---------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|------------------|
+| Local        | The local file system.                                        | (none)                                                                                                                   | 0.5.0            |
+| HDFS         | HDFS file system.                                             | (none)                                                                                                                   | 0.5.0            |
+| S3           | AWS S3.                                                       | [gravitino-aws-hadoop-bundle](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-hadoop-aws-bundle)       | 0.8.0-incubating |
+| GCS          | Google Cloud Storage.                                         | [gravitino-gcp-hadoop-bundle](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-hadoop-gcp-bundle)       | 0.8.0-incubating |
+| OSS          | Aliyun OSS.                                                   | [gravitino-aliyun-hadoop-bundle](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-hadoop-aliyun-bundle) | 0.8.0-incubating |
+| ABS          | Azure Blob Storage (aka. ABS, or Azure Data Lake Storage (v2) | [gravitino-azure-hadoop-bundle](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-hadoop-azure-bundle)   | 0.8.0-incubating |
+
+After adding the jars into the fileset catalog classpath, you can start up the Gravitino server by running the following command:
+
+```shell
+cd ${GRAVITINO_HOME}
+bin/gravitino.sh start
+```
+
+### Bundle jars
+
+Gravitino bundles jars are jars that are used to access the cloud storage, they are divided into two categories:
```

Comment:
We have two sentences here... and the wordy statement can be simplified without losing anything:

```suggestion
Gravitino bundles jars are used to access the cloud storage.
They are divided into two categories:
```